src/models/backingImage.js

```diff
@@ -4,6 +4,7 @@ import { wsChanges, updateState } from '../utils/websocket'
 import queryString from 'query-string'
 
 export default {
+  ws: null,
```

Comment:
Should ws be placed in `state`?

Reply:
Yes, we need to record the ws, otherwise, we don’t know which ws to close when we stop the ws